site/content/en/docs/Installation/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Installation"
+linkTitle: "Installation"
+weight: 3
+---
+
+Installing the AWS Copilot CLI currently requires you to download our binary from the GitHub releases page manually. In the future, we'll distribute the binary through homebrew and other binaries as well.
+
+In the meantime, to install, copy and paste the command into your terminal.
+
+| Platform | Command to install |
+|---------|---------
+| macOS | `curl -Lo /usr/local/bin/copilot https://github.com/aws/copilot-cli/releases/download/v0.1.0/copilot-darwin-v0.1.0 && chmod +x /usr/local/bin/copilot && copilot --help` |
+| Linux | `curl -Lo /usr/local/bin/copilot https://github.com/aws/copilot-cli/releases/download/v0.1.0/copilot-linux-v0.1.0 && chmod +x /usr/local/bin/copilot && copilot --help` |
```

Comment:
How about our windows friends🤔

Reply:
I don't think we're listing the windows binary for now as we haven't tested the binary in a while :P

Reply:
i wonder how many users are using Windows...maybe we should put windows support on our calendar if they are a considerable amount of them

Reply:
I was wondering about the windows version of this cli. I am new to containers and the demo you guys did so far on how to push containers to ECS  is straightforward  but I couldn't find how to install  this for my win machine.

Reply:
> I was wondering about the windows version of this cli. I am new to containers and the demo you guys did so far on how to push containers to ECS is straightforward but I couldn't find how to install this for my win machine.

Hi @RoniqueRicketts! We do provide a windows binary under our [releases](https://github.com/aws/copilot-cli/releases/tag/v0.1.0): [copilot-windows-v0.1.0.exe](https://github.com/aws/copilot-cli/releases/download/v0.1.0/copilot-windows-v0.1.0.exe) so you should be able to run the executable. However, we haven't tested the experience with the windows binary for a while. 

Feel free to open an issue if you experience difficulties with the binary!